packages/ng/libraries/core/src/lib/option/picker/option-picker.component.html

```diff
@@ -1,6 +1,6 @@
-<ng-template>
+<ng-template class="option-picker">
 	<div class="lu-popover-panel" role="dialog" [ngClass]="panelClassesMap" [class.mod-multiple]="multiple"

```

Comment:
the calss should be on this div, not its parent

the tag `ng-template` is a virtual tag used by angular, it will never be present and rendererd in the actual DOM